blog/13-12-22-layer-1-and-layer-2/Layer-1-and-layer-2.md

```diff
@@ -0,0 +1,87 @@
+---
+title: An Introduction to Layer 1 and Layer 2 Blockchain Protocols
+description: In this tutorial, we will provide an introduction to the different types of blockchain protocols and explain the key differences between layer 1 and layer 2 protocols. We will discuss the benefits and limitations of each type of protocol, as well as provide examples of how they are used in the real world. By the end of this tutorial, you will have a better understanding of the different types of blockchain protocols and how they are used in web3 development. This tutorial is essential for anyone looking to learn about blockchain protocols and how they impact the development of decentralized applications.
+authors:
+  - name: ✍️ Obafemi Joshua 
+tags: [celo]
+hide_table_of_contents: true
+slug: "/tutorials/An-Introduction-to-Layer-1-and-Layer-2-Blockchain-Protocols"
+---
+
+
+#Layer 1 and Layer 2 Protocols.
+blog\13-12-22-layer-1-and-layer-2\images\img1 .png
```

Comment:
Please add the image in markdown format.
https://marinegeo.github.io/2018-08-10-adding-images-markdown/